tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -144,3 +146,12 @@ def tap_import_certs(self):
         Wait(self.marionette).until(expected.element_displayed(element))
         element.tap()
         Wait(self.marionette).until(expected.element_displayed(*self._select_certs_screen_locator))
+    
+    def tap_active_wifi(self):
```

Comment:
We decided to not use UI action names for methods, but more describing what the result of the UI action is, so in this case open_active_wifi would be better.